settings.nix

```diff
@@ -1192,6 +1192,24 @@ with docs.lib; {
                 This will also set the XCURSOR_SIZE environment variable for all spawned processes.
               '';
             };
+          hide-on-key-press =
+            optional types.bool false
+            // {
+              description = ''
+                ${unstable-note}
+
+                If set, hides the cursor when pressing a key on the keyboard.
```

Comment:
This is copied from niri wiki, yea? In that case, this node is either missing or "set". In our case, there's a slight semantic difference in that it is always "present" and set to false or true; there is no "unset" state for this option, and i think the wording ought to reflect that
```suggestion
                Whether to hide the cursor when a key is pressed on the keyboard.
```